src/UnscentedTransformKalmanInversion.jl

```diff
@@ -0,0 +1,474 @@
+#Unscented Transform Kalman Inversion: specific structures and function definitions
+
+# Many functions are shared with Unscented Kalman Inversion, and are defined in `src/UnscentedKalmanInversion.jl`
+"""
+    TransformUnscented{FT<:AbstractFloat, IT<:Int} <: Process
+
+An unscented Kalman Inversion process.
+
+# Fields
+
+$(TYPEDFIELDS)
+
+# Constructors
+
+    TransformUnscented(
+        u0_mean::AbstractVector{FT},
+        uu0_cov::AbstractMatrix{FT};
+        α_reg::FT = 1.0,
+        update_freq::IT = 0,
+        modified_unscented_transform::Bool = true,
+        impose_prior::Bool = false,
+        prior_mean::Any,
+        prior_cov::Any,
+        sigma_points::String = symmetric
+    ) where {FT <: AbstractFloat, IT <: Int}
+
+Construct an TransformUnscented Inversion Process.
+
+Inputs:
+
+  - `u0_mean`: Mean at initialization.
+  - `uu0_cov`: Covariance at initialization.
+  - `α_reg`: Hyperparameter controlling regularization toward the prior mean (0 < `α_reg` ≤ 1),
+  default should be 1, without regulariazion.
```

Comment:
Typo "regulariazion". Also, maybe change to "default is 1"?

Reply:
Done